### ISSUE TITLE
Broken `cargo install`

### ISSUE THREAD

**Issue by Skgland** [langston-barrett/icemelter#55]
```
~> cargo --version 
cargo 1.69.0 (6e9a83356 2023-04-12)
```

<details>
<summary>Log</summary>

```shell

~> cargo install icemelter
    Updating crates.io index
  Installing icemelter v0.3.1
   Compiling proc-macro2 v1.0.58
   Compiling quote v1.0.27
   Compiling unicode-ident v1.0.9
   Compiling libc v0.2.144
   Compiling io-lifetimes v1.0.11
   Compiling once_cell v1.17.1
   Compiling memchr v2.5.0
   Compiling rustix v0.37.19
   Compiling cc v1.0.79
   Compiling cfg-if v1.0.0
   Compiling linux-raw-sys v0.3.8
   Compiling bitflags v1.3.2
   Compiling regex-syntax v0.7.2
   Compiling log v0.4.17
   Compiling serde v1.0.163
   Compiling tracing-core v0.1.31
   Compiling utf8parse v0.2.1
   Compiling anstyle-parse v0.2.0
   Compiling colorchoice v1.0.0
   Compiling anstyle v1.0.0
   Compiling serde_json v1.0.96
   Compiling anstyle-query v1.0.0
   Compiling clap_lex v0.5.0
   Compiling thiserror v1.0.40
   Compiling heck v0.4.1
   Compiling aho-corasick v1.0.1
   Compiling lazy_static v1.4.0
   Compiling ryu v1.0.13
   Compiling itoa v1.0.6
   Compiling strsim v0.10.0
   Compiling syn v2.0.16
   Compiling pin-project-lite v0.2.9
   Compiling anyhow v1.0.71
   Compiling fastrand v1.9.0
   Compiling overload v0.1.1
   Compiling nu-ansi-term v0.46.0
   Compiling tree-sitter v0.20.10
   Compiling tree-sitter-rust v0.20.3
   Compiling tracing-log v0.1.3
   Compiling sharded-slab v0.1.4
   Compiling regex v1.8.3
   Compiling wait-timeout v0.2.0
   Compiling thread_local v1.1.7
   Compiling smallvec v1.10.0
   Compiling num_cpus v1.15.0
   Compiling is-terminal v0.4.7
   Compiling anstream v0.3.2
   Compiling clap_builder v4.3.0
   Compiling tempfile v3.5.0
   Compiling tree-sitter-traversal v0.1.2
   Compiling tree-sitter-edit v0.3.0
   Compiling serde_derive v1.0.163
   Compiling tracing-attributes v0.1.24
   Compiling clap_derive v4.3.0
   Compiling thiserror-impl v1.0.40
   Compiling tracing v0.1.37
   Compiling clap v4.3.0
   Compiling clap-verbosity-flag v2.0.1
   Compiling tracing-serde v0.1.3
   Compiling treereduce v0.2.2
   Compiling tracing-subscriber v0.3.17
   Compiling icemelter v0.3.1
error[E0063]: missing field `delete_non_optional` in initializer of `Config<_>`
   --> /home/bennet/.cargo/registry/src/github.com-1ecc6299db9ec823/icemelter-0.3.1/src/main.rs:264:25
    |
264 |     let reduce_config = Config {
    |                         ^^^^^^ missing `delete_non_optional`

For more information about this error, try `rustc --explain E0063`.
error: could not compile `icemelter` due to previous error
error: failed to compile `icemelter v0.3.1`, intermediate artifacts can be found at `/tmp/cargo-installv3d3mT`

```

</details>

**Comment by langston-barrett** [comment]
Thanks for the report! In the short term, you can try installing from the git repo:
```sh
git clone https://github.com/langston-barrett/icemelter
cd icemelter
cargo install --path .
```

**Comment by langston-barrett** [comment]
Fixed in v0.3.2
